system_metrics_collector/src/system_metrics_collector/periodic_measurement_node.cpp

```diff
@@ -33,40 +33,40 @@ PeriodicMeasurementNode::PeriodicMeasurementNode(
   const std::chrono::milliseconds publish_period,
   const bool clear_measurements_on_publish)
 : Node(name),
-  measurement_period_(measurement_period),
   publishing_topic_(publishing_topic),
-  measurement_timer_(nullptr),
```

Comment:
Why move initialization?

Reply:
No longer changing order of member variable initialization in this PR.